src/cmd.fs

```diff
@@ -41,76 +37,203 @@ module Cmd =
     let batch (cmds: #seq<Cmd<'msg>>) : Cmd<'msg> =
         cmds |> List.concat
 
-    /// Command that will evaluate an async block and map the result
-    /// into success or error (of exception)
-    let ofAsync (task: 'a -> Async<_>)
-                (arg: 'a)
-                (ofSuccess: _ -> 'msg)
-                (ofError: _ -> 'msg) : Cmd<'msg> =
-        let bind dispatch =
-            async {
-                let! r = task arg |> Async.Catch
-                dispatch (match r with
-                         | Choice1Of2 x -> ofSuccess x
-                         | Choice2Of2 x -> ofError x)
-            }
-        [bind >> Async.StartImmediate]
-
-    /// Command to evaluate a simple function and map the result
-    /// into success or error (of exception)
-    let ofFunc (task: 'a -> _) (arg: 'a) (ofSuccess: _ -> 'msg) (ofError: _ -> 'msg) : Cmd<'msg> =
-        let bind dispatch =
-            try
-                task arg
-                |> (ofSuccess >> dispatch)
-            with x ->
-                x |> (ofError >> dispatch)
-        [bind]
-
-    /// Command to evaluate a simple function and map the success to a message
-    /// discarding any possible error
-    let performFunc (task: 'a -> _) (arg: 'a) (ofSuccess: _ -> 'msg) : Cmd<'msg> =
-        let bind dispatch =
-            try
-                task arg
-                |> (ofSuccess >> dispatch)
-            with x ->
-                ()
-        [bind]
-
-    /// Command to evaluate a simple function and map the error (in case of exception)
-    let attemptFunc (task: 'a -> unit) (arg: 'a) (ofError: _ -> 'msg) : Cmd<'msg> =
-        let bind dispatch =
-            try
-                task arg
-            with x ->
-                x |> (ofError >> dispatch)
-        [bind]
-
     /// Command to call the subscriber
     let ofSub (sub: Sub<'msg>) : Cmd<'msg> =
         [sub]
 
+    module OfFunc =
```

Comment:
Is `Of` adding value compare to naming the module `Func`?

Reply:
I think so - it is more discoverable by people who just start typing the same thing they are used to, and I'd want it in the name - function or module, the module seems preferable.

Reply:
Ok, thanks for the explanations.